src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -124,7 +125,9 @@ case class Bagger(
     val helper = BaggerHelper(models, trainingData, Nib, useJackknife, uncertaintyCalibration)
     val biasModel = if (biasLearner.isDefined && helper.oobErrors.nonEmpty && helper.isRegression) {
       Async.canStop()
```

Comment:
Is this necessary anymore? Is the `Async` object necessary at all anymore?

Reply:
No. I'm going to pull out `ASync` in a follow-up PR. There are several things I want to do before we release v5 😄